Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -382,19 +382,25 @@ extension GridTabViewController {
         }
     }
 
-    func closeTabsForCurrentTray() {
-        let tabs = self.tabDisplayManager.isPrivate ? tabManager.privateTabs : tabManager.normalTabs
-        let maxTabs = 100
-        if self.tabDisplayManager.isPrivate {
-            let previous = self.tabManager.selectedTab
-            self.tabManager.removeTabsWithoutToast(tabs)
-            self.tabManager.selectTab(mostRecentTab(inTabs: tabManager.normalTabs) ?? tabManager.normalTabs.last, previous: previous)
-        } else if tabs.count >= maxTabs {
-            self.tabManager.removeTabsAndAddNormalTab(tabs)
-        } else {
-            self.tabManager.removeTabsWithToast(tabs)
+    func closeTabsTrayBackground() {
+        tabDisplayManager.removeAllTabs()
```

Comment:
Can we rename `removeAllTabs`? It's confusing, because, afterwards, you're `backgroundRemoveAllTabs`. Let's call it something reflective of what it's doing, like, `removeAllTabsFromUI` or something.